terraform/app/modules/paas/main.tf

```diff
@@ -18,6 +18,11 @@ resource cloudfoundry_app web_app {
   stopped = var.app_stopped
   strategy = var.web_app_deployment_strategy
   timeout = var.app_start_timeout
+
+  service_binding {
+    service_instance = cloudfoundry_user_provided_service.logging.id
+  }
+
```

Comment:
We have dynamic service binding below. I wonder if they could be put together.

Reply:
This would involve adding `cloudfoundry_user_provided_service.logging.id` to `app_cloudfoundry_service_instances` on line 54 of terraform/app/modules/paas/variables.tf, and removing this service_binding